Gemfile

```diff
@@ -19,3 +19,8 @@ group :development, :test do
   gem 'pry'
   gem 'generator_spec'
 end
+
+group :test do
+  gem 'test-unit'
```

Comment:
@domaindrivendev I investigated a little more. It originates from gem generator_spec which depends on railties which depends on this gem.
Also I found:
"Ruby 2.2+ has removed test/unit from the core library. Rails requires this as a dependency."
And I am using ruby 2.3